### ISSUE TITLE
Question: FormBuilderDropDown appearance

### ISSUE THREAD

**Issue by mattf80** [danvick/flutter_form_builder#113]
Apologies if this has already been asked/is covered by the docs, I could not find the answer. 

My question is how to change the appearance of the FormBuilderDropDown button to use the 'outline' style. It seems to default to the 'underline' style and I cannot figure out how to change this.  It does not seem to respect the app-wide theme I have applied:

```
    inputDecorationTheme: InputDecorationTheme(
      border: OutlineInputBorder(),
    ),
```
even though the other types of FormBuilder inputs do (i.e FormBuilderTextField). I also tried setting it directly on the widget itself:

```
          FormBuilder(
            key: _fbKey,
            child: Column(
              children: <Widget>[
                FormBuilderDropdown(
                    decoration: InputDecoration(
                      border: OutlineInputBorder(
                        borderSide: BorderSide(color: Colors.red),
                      ),
                    ),
                    attribute: 'woo',
                    items: ['Tommy']
                        .map(
                          (val) => DropdownMenuItem(
                            value: val,
                            child: Text(val),
                          ),
                        )
                        .toList()),
              ],
            ),
          ),
```

But again no joy. Any help would be much appreciated - thanks!

**Comment by danvick** [comment]
Hi @mattf80,
Thanks for bringing this to my attention. 
A fix for this is in the works and will be published as soon as possible. I'll let you know when published.

**Comment by danvick** [comment]
Kindly confirm that this issue is addressed in `v3.4.0`. 

Feel free to reopen the issue if not.

**Comment by mattf80** [comment]
Hi @danvick confirming this is fixed. Thanks so much!
